Tests/Data/Collections/Util/Array2DExtensionTests.cs

```diff
@@ -0,0 +1,94 @@
+using NUnit.Framework;
+using System;
+using Anvil.CSharp.Data;
+
+namespace Anvil.CSharp.Tests
+{
+    public static class Array2DExtensionTests
+    {
+        [Test, Order(0)]
```

Comment:
This `[Order]` attribute just ensures the `PopulateTest` runs before the rest, as most depend on that function. Tests without this attribute always run after those that do.